src/instructlab/model/serve.py

```diff
@@ -57,6 +57,16 @@
     required=False,
     help="Log file path to write server logs to.",
 )
+@click.option(
+    "--chat-template",
+    type=str,
```

Comment:
After looking at and addressing https://github.com/instructlab/instructlab/pull/1636/files#r1677217137, would we want to consider setting the default here to `tokenizer just so that `ilab model serve` with granite and merlinite out of the box just works`?